controllers/build.go

```diff
@@ -3,11 +3,11 @@ package controllers
 import (
 	goerrors "errors"
 	"fmt"
+	"github.com/openfunction/pkg/util"
```

Comment:
I think we should use the `Goimports` to format code, not  `Go Fmt`.

Reply:
Have formatted the code by using `Goimports`.

Reply:
And add `goimports` commond in Makefile